pipit/writers/chrome_writer.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+
+class ChromeWriter:
+    """Exports traces to Chrome Tracing JSON format which can be opened with Chrome Trace Viewer and Perfetto.
+
+    See https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview # noqa
+    """
+
+    def __init__(self, trace, filename=None):
+        self.trace = trace
+        self.filename = filename
+
+    def convert(self):
+        events = self.trace.events
+
+        df = pd.DataFrame()
+        df["name"] = events["Name"]
+        df["ph"] = events["Event Type"].replace(
+            ["Enter", "Leave", "Instant"], ["B", "E", "i"]
+        )
+        df["ts"] = (events["Timestamp (ns)"] / 1e3).astype(int)
```

Comment:
I'm assuming this is going off the default time unit in the document which is ms. But aren't there 1e6 ns in 1 ms?